release.py

```diff
@@ -156,10 +156,23 @@ def _create_gh_release(payload, token):
 def _upload_assets(upload_url, token):
     assert token is not None, 'Need token for this action'
     import requests
-    for wheelpath in ['build', 'dist', 'wheelhouse']:
-        wheelfile = os.path.basename(wheelpath)
-        print(f'Uploading wheel {wheelpath}')
-        with open(wheelpath, 'rb') as f:
+    if os.path.exists('wheelhouse'):
+        wheelpaths = [os.path.join('wheelhouse', ff) for ff in os.listdir('wheelhouse')]
+    if wheelpaths is not None:
```

Comment:
If `wheelpaths is None` obviously the wheels can't be uploaded - but would this still pass the CI?
Also are there wheels in all these directories (which is what it looks like from the code)? Or is it that depending on the OS it could be in any of these?

Reply:
Actually I think there's a bug here... I've not actually tested running a release because the CI is a pain and I didn't want to make a release yet, but you're right I think that this bit of the code won't work (it was written in a bit of a hurry just before the training course). So different programs use different default folders (`dist` or `wheelhouse`) and actually in our case I specify in the yaml to use `build` ([line 122 of build_pyspinw.yml](https://github.com/SpinW/spinw/blob/master/.github/workflows/build_pyspinw.yml#L122))... so it should be looking in there... I think I'll try to change the code to use `glob` or something instead.